Unable to deploy to iOS 15.4 emulator

I am experiencing the same issue as described here https://github.com/xamarin/xamarin-macios/issues/14560#issuecomment-1100645415 by this user. 

It appears that the main issue was closed, but I still have the issue after installing Xamarin.iOS Version 15.8.0.4 

This is the only line which appears in the Application Output:
`error HE0042: Could not launch the app 'skydrop.skydrop' on the device 'iOS 15.4 (19E240) - iPhone 13 Pro': Application launch for 'skydrop.skydrop' did not return a valid pid nor a launch error.`


### Steps to Reproduce

1.  Pull repo of Xamarin native app which was initially created in VS For Mac 2019
2. Open solution in VS For Mac 2022 (Preview RC)
3. Try debug app on iPhone 15.4 emulator

### Expected Behavior
App builds and deploys, debugger is started as normal.

### Actual Behavior
App builds and deploys, but debugger fails to start. 

### Environment

```
Visual Studio Community 2022 for Mac Preview
Version 17.0 Preview (17.0 build 8754)
Installation UUID: d5a91729-631e-430f-984e-d831b5006fd5

Runtime
.NET 6.0.3 (64-bit)
Architecture: Arm64

Roslyn (Language Service)
4.1.0-3.22075.3+592501cbb9c9394072a245c15b3458ff88155d85

NuGet
Version: 6.0.0.262

.NET SDK (Arm64)
SDK: /usr/local/share/dotnet/sdk/6.0.202/Sdks
SDK Versions:
	6.0.202
	6.0.201
	6.0.101
MSBuild SDKs: /usr/local/share/dotnet/sdk/6.0.202/Sdks

.NET SDK (x64)
SDK Versions:
	6.0.202
	6.0.201
	6.0.103
	6.0.102
	6.0.101
	5.0.407
	5.0.406
	5.0.405
	5.0.404
	3.1.418
	3.1.417
	3.1.416

.NET Runtime (Arm64)
Runtime: /usr/local/share/dotnet/dotnet
Runtime Versions:
	6.0.4
	6.0.3
	6.0.1

.NET Runtime (x64)
Runtime: /usr/local/share/dotnet/x64/dotnet
Runtime Versions:
	6.0.4
	6.0.3
	6.0.2
	6.0.1
	5.0.16
	5.0.15
	5.0.14
	5.0.13
	3.1.24
	3.1.23
	3.1.22

Xamarin.Profiler
Version: 1.8.0.19
Location: /Applications/Xamarin Profiler.app/Contents/MacOS/Xamarin Profiler

Updater
Version: 11

Apple Developer Tools
Xcode 13.3.1 (20103)
Build 13E500a

Xamarin.Mac
Version: 8.8.0.3 (Visual Studio Community)
Hash: eb7f8ed35
Branch: xcode13.3
Build date: 2022-03-30 03:16:09-0400

Xamarin.iOS
Version: 15.8.0.4 (Visual Studio Community)
Hash: f2005bab4
Branch: xcode13.3
Build date: 2022-03-31 02:23:58-0400

Xamarin Designer
Version: 17.1.5.88
Hash: 25a718e34
Branch: remotes/origin/d17-0-vsmac
Build date: 2022-04-22 14:55:54 UTC

Xamarin.Android
Version: 12.3.0.3 (Visual Studio Community)
Commit: xamarin-android/d17-2/bbba5a2
Android SDK: /Users/jh/Library/Developer/Xamarin/android-sdk-macosx
	Supported Android versions:
		12.0 (API level 31)
		11.0 (API level 30)
		10.0 (API level 29)

SDK Command-line Tools Version: 5.0
SDK Platform Tools Version: 31.0.3
SDK Build Tools Version: 30.0.3

Build Information: 
Mono: adf1bc4
Java.Interop: xamarin/java.interop/d17-2@9760f0a9
ProGuard: Guardsquare/proguard/v7.0.1@912d149
SQLite: xamarin/sqlite/3.38.2@7b1e016
Xamarin.Android Tools: xamarin/xamarin-android-tools/d17-2@fc3c2ac

Microsoft Build of OpenJDK
Java SDK: /Library/Java/JavaVirtualMachines/microsoft-11.jdk
11.0.12
Android Designer EPL code available here:
https://github.com/xamarin/AndroidDesigner.EPL

Eclipse Temurin JDK
Java SDK: /Library/Java/JavaVirtualMachines/temurin-8.jdk
1.8.0.302
Android Designer EPL code available here:
https://github.com/xamarin/AndroidDesigner.EPL

Android SDK Manager
Version: 17.2.0.32
Hash: 358904c
Branch: remotes/origin/d17-2
Build date: 2022-04-22 14:55:59 UTC

Android Device Manager
Version: 0.0.0.1110
Hash: 6cb643c
Branch: remotes/origin/d17-2
Build date: 2022-04-22 14:55:59 UTC

Build Information
Release ID: 1700008754
Git revision: b1c3df15b587d32846f3254e47d654dfb6d3627b
Build date: 2022-04-22 14:54:05+00
Build branch: release-17.0
Build lane: release-17.0

Operating System
Mac OS X 12.3.1
Darwin 21.4.0 Darwin Kernel Version 21.4.0
    Fri Mar 18 00:47:26 PDT 2022
    root:xnu-8020.101.4~15/RELEASE_ARM64_T8101 arm64


```



### Build Logs

```

Building SkyDrop.iOS (Debug|iPhoneSimulator)
Build started 27/04/2022 13:34:34.
__________________________________________________
Project "/Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/SkyDrop.Core.csproj" (Build target(s)):

Target ResolvePackageAssets:
    /Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/SkyDrop.Core.csproj : warning NU1701: Package 'Microsoft.Bcl 1.1.10' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
    /Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/SkyDrop.Core.csproj : warning NU1701: Package 'Microsoft.Net.Http 2.2.29' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
Done building target "ResolvePackageAssets" in project "SkyDrop.Core.csproj".
Target GenerateTargetFrameworkMonikerAttribute:
  Skipping target "GenerateTargetFrameworkMonikerAttribute" because all output files are up-to-date with respect to the input files.
Target CoreGenerateAssemblyInfo:
  Skipping target "CoreGenerateAssemblyInfo" because all output files are up-to-date with respect to the input files.
Target CoreCompile:
  Skipping target "CoreCompile" because all output files are up-to-date with respect to the input files.
Target FodyTarget:
  Skipping target "FodyTarget" because all output files are up-to-date with respect to the input files.
Target GenerateBuildDependencyFile:
  Skipping target "GenerateBuildDependencyFile" because all output files are up-to-date with respect to the input files.
Target CopyFilesToOutputDirectory:
    SkyDrop.Core -> /Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/bin/Debug/netstandard2.0/SkyDrop.Core.dll

Done building project "SkyDrop.Core.csproj".
__________________________________________________
Project "/Users/jh/Documents/source/SkyDrop/src/SkyDrop.iOS/SkyDrop.iOS.csproj" (Build target(s)):

Target _DetectDebugNetworkConfiguration:
      DebugIPAddresses: 127.0.0.1
Target _BeforeCoreCompileImageAssets:
  Skipping target "_BeforeCoreCompileImageAssets" because all output files are up-to-date with respect to the input files.
Target _DetectSigningIdentity:
    Detected signing identity:
            
      Bundle Id: skydrop.skydrop
      App Id: skydrop.skydrop
Target _CoreCompileImageAssets:
  Skipping target "_CoreCompileImageAssets" because all output files are up-to-date with respect to the input files.
Target _BeforeCoreCompileInterfaceDefinitions:
  Skipping target "_BeforeCoreCompileInterfaceDefinitions" because all output files are up-to-date with respect to the input files.
Target _CoreCompileInterfaceDefinitions:
  Skipping target "_CoreCompileInterfaceDefinitions" because all output files are up-to-date with respect to the input files.
Target _BeforeCoreCompileSceneKitAssets:
  Skipping target "_BeforeCoreCompileSceneKitAssets" because it has no inputs.
Target _BeforeCoreCompileSceneKitAssets:
  Skipping target "_BeforeCoreCompileSceneKitAssets" because it has no inputs.
Target _BeforeCoreCompileSceneKitAssets:
  Skipping target "_BeforeCoreCompileSceneKitAssets" because it has no inputs.
Target _CoreCompileSceneKitAssets:
  Skipping target "_CoreCompileSceneKitAssets" because it has no inputs.
Target _CoreCompileColladaAssets:
  Skipping target "_CoreCompileColladaAssets" because it has no inputs.
Target _BeforeCompileTextureAtlases:
  Skipping target "_BeforeCompileTextureAtlases" because it has no inputs.
Target _BeforeCompileTextureAtlases:
  Skipping target "_BeforeCompileTextureAtlases" because it has no inputs.
Target _BeforeCompileTextureAtlases:
  Skipping target "_BeforeCompileTextureAtlases" because it has no inputs.
Target _CoreCompileTextureAtlases:
  Skipping target "_CoreCompileTextureAtlases" because it has no inputs.
Target _BeforeCompileCoreMLModels:
  Skipping target "_BeforeCompileCoreMLModels" because all output files are up-to-date with respect to the input files.
Target _CoreCompileCoreMLModels:
  Skipping target "_CoreCompileCoreMLModels" because all output files are up-to-date with respect to the input files.
Target _CoreOptimizePngImages:
  Skipping target "_CoreOptimizePngImages" because it has no outputs.
Target _CoreOptimizePropertyLists:
  Skipping target "_CoreOptimizePropertyLists" because all output files are up-to-date with respect to the input files.
Target _CoreOptimizeLocalizationFiles:
  Skipping target "_CoreOptimizeLocalizationFiles" because it has no inputs.
Target _GetProjectReferenceTargetFrameworkProperties:
    __________________________________________________
    Project "/Users/jh/Documents/source/SkyDrop/src/SkyDrop.iOS/SkyDrop.iOS.csproj" is building "/Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/SkyDrop.Core.csproj" (GetTargetFrameworks target(s)):

Target ResolveProjectReferences:
    __________________________________________________
    Project "/Users/jh/Documents/source/SkyDrop/src/SkyDrop.iOS/SkyDrop.iOS.csproj" is building "/Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/SkyDrop.Core.csproj" (GetTargetPath target(s)):

    __________________________________________________
    Project "/Users/jh/Documents/source/SkyDrop/src/SkyDrop.iOS/SkyDrop.iOS.csproj" is building "/Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/SkyDrop.Core.csproj" (GetNativeManifest target(s)):

Target ResolveAssemblyReferences:
    /Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277: Found conflicts between different versions of "Microsoft.CSharp" that could not be resolved.
    /Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277: There was a conflict between "Microsoft.CSharp, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a".
    /Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:     "Microsoft.CSharp, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was chosen because it was primary and "Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was not.
    /Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:     References which depend on "Microsoft.CSharp, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Microsoft.CSharp.dll].
    /Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:         /Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Microsoft.CSharp.dll
    /Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:           Project file item includes which caused reference "/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Microsoft.CSharp.dll".
    /Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:             Microsoft.CSharp
    /Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:     References which depend on "Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [].
    /Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:         /Users/jh/.nuget/packages/realm/10.1.2/lib/netstandard2.0/Realm.dll
    /Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:           Project file item includes which caused reference "/Users/jh/.nuget/packages/realm/10.1.2/lib/netstandard2.0/Realm.dll".
    /Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:             /Users/jh/.nuget/packages/realm/10.1.2/lib/netstandard2.0/Realm.dll
    /Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:             /Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/bin/Debug/netstandard2.0/SkyDrop.Core.dll
    Consider app.config remapping of assembly "System.Net.Http, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "1.5.0.0" [] to Version "4.0.0.0" [/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/System.Net.Http.dll] to solve conflict and get rid of warning.
    Consider app.config remapping of assembly "System.Buffers, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" from Version "4.0.2.0" [] to Version "4.0.99.0" [/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Facades/System.Buffers.dll] to solve conflict and get rid of warning.
    Consider app.config remapping of assembly "netstandard, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" from Version "2.0.0.0" [] to Version "2.1.0.0" [/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Facades/netstandard.dll] to solve conflict and get rid of warning.
    Consider app.config remapping of assembly "System.Runtime, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.20.0" [] to Version "4.1.0.0" [/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Facades/System.Runtime.dll] to solve conflict and get rid of warning.
    Consider app.config remapping of assembly "System.Collections, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Facades/System.Collections.dll] to solve conflict and get rid of warning.
    Consider app.config remapping of assembly "System.Runtime.Extensions, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.10.0" [] to Version "4.1.0.0" [/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Facades/System.Runtime.Extensions.dll] to solve conflict and get rid of warning.
    Consider app.config remapping of assembly "System.Linq, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.1.0.0" [/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Facades/System.Linq.dll] to solve conflict and get rid of warning.
    Consider app.config remapping of assembly "System.Threading, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Facades/System.Threading.dll] to solve conflict and get rid of warning.
    Consider app.config remapping of assembly "System.IO, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.10.0" [] to Version "4.1.0.0" [/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Facades/System.IO.dll] to solve conflict and get rid of warning.
    Consider app.config remapping of assembly "System.Text.Encoding, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.0.10.0" [/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Facades/System.Text.Encoding.dll] to solve conflict and get rid of warning.
    Consider app.config remapping of assembly "System.Text.RegularExpressions, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" from Version "4.0.0.0" [] to Version "4.1.0.0" [/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Facades/System.Text.RegularExpressions.dll] to solve conflict and get rid of warning.
    /Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3276: Found conflicts between different versions of the same dependent assembly. Please set the "AutoGenerateBindingRedirects" property to true in the project file. For more information, see http://go.microsoft.com/fwlink/?LinkId=294190.
Done building target "ResolveAssemblyReferences" in project "SkyDrop.iOS.csproj".
Target GenerateTargetFrameworkMonikerAttribute:
  Skipping target "GenerateTargetFrameworkMonikerAttribute" because all output files are up-to-date with respect to the input files.
Target CoreCompile:
  Skipping target "CoreCompile" because all output files are up-to-date with respect to the input files.
Target FodyTarget:
  Skipping target "FodyTarget" because all output files are up-to-date with respect to the input files.
Target _CopyFilesMarkedCopyLocal:
    Touching "/Users/jh/Documents/source/SkyDrop/src/SkyDrop.iOS/obj/iPhoneSimulator/Debug/SkyDrop.iOS.csproj.CopyComplete".
Target GetCopyToOutputDirectoryItems:
    Target _GetCopyToOutputDirectoryItemsFromTransitiveProjectReferences:
        __________________________________________________
        Project "/Users/jh/Documents/source/SkyDrop/src/SkyDrop.iOS/SkyDrop.iOS.csproj" is building "/Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/SkyDrop.Core.csproj" (GetCopyToOutputDirectoryItems target(s)):

        Target ResolvePackageAssets:
            /Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/SkyDrop.Core.csproj : warning NU1701: Package 'Microsoft.Bcl 1.1.10' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
            /Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/SkyDrop.Core.csproj : warning NU1701: Package 'Microsoft.Net.Http 2.2.29' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
        Done building target "ResolvePackageAssets" in project "SkyDrop.Core.csproj".

        Done building project "SkyDrop.Core.csproj".
Target _CopyOutOfDateSourceItemsToOutputDirectory:
  Skipping target "_CopyOutOfDateSourceItemsToOutputDirectory" because all output files are up-to-date with respect to the input files.
Target CopyFilesToOutputDirectory:
    SkyDrop.iOS -> /Users/jh/Documents/source/SkyDrop/src/SkyDrop.iOS/bin/iPhoneSimulator/Debug/SkyDrop.iOS.exe
Target _CopyResourcesToBundle:
  Skipping target "_CopyResourcesToBundle" because all output files are up-to-date with respect to the input files.
Target _WriteAppManifest:
  Skipping target "_WriteAppManifest" because all output files are up-to-date with respect to the input files.
Target _CompileToNative:
  Skipping target "_CompileToNative" because all output files are up-to-date with respect to the input files.
Target _CopyAppExtensionsToBundle:
  Skipping target "_CopyAppExtensionsToBundle" because it has no inputs.

Done building project "SkyDrop.iOS.csproj".

Build succeeded.

/Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/SkyDrop.Core.csproj : warning NU1701: Package 'Microsoft.Bcl 1.1.10' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
/Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/SkyDrop.Core.csproj : warning NU1701: Package 'Microsoft.Net.Http 2.2.29' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
/Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277: Found conflicts between different versions of "Microsoft.CSharp" that could not be resolved.
/Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277: There was a conflict between "Microsoft.CSharp, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a".
/Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:     "Microsoft.CSharp, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was chosen because it was primary and "Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was not.
/Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:     References which depend on "Microsoft.CSharp, Version=2.0.5.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Microsoft.CSharp.dll].
/Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:         /Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Microsoft.CSharp.dll
/Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:           Project file item includes which caused reference "/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/mono/Xamarin.iOS/Microsoft.CSharp.dll".
/Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:             Microsoft.CSharp
/Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:     References which depend on "Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [].
/Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:         /Users/jh/.nuget/packages/realm/10.1.2/lib/netstandard2.0/Realm.dll
/Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:           Project file item includes which caused reference "/Users/jh/.nuget/packages/realm/10.1.2/lib/netstandard2.0/Realm.dll".
/Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:             /Users/jh/.nuget/packages/realm/10.1.2/lib/netstandard2.0/Realm.dll
/Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3277:             /Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/bin/Debug/netstandard2.0/SkyDrop.Core.dll
/Applications/Visual Studio (Preview).app/Contents/MonoBundle/MSBuild/Current/bin/Microsoft.Common.CurrentVersion.targets(2218,5): warning MSB3276: Found conflicts between different versions of the same dependent assembly. Please set the "AutoGenerateBindingRedirects" property to true in the project file. For more information, see http://go.microsoft.com/fwlink/?LinkId=294190.
/Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/SkyDrop.Core.csproj : warning NU1701: Package 'Microsoft.Bcl 1.1.10' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
/Users/jh/Documents/source/SkyDrop/src/SkyDrop.Core/SkyDrop.Core.csproj : warning NU1701: Package 'Microsoft.Net.Http 2.2.29' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETStandard,Version=v2.0'. This package may not be fully compatible with your project.
    6 Warning(s)
    0 Error(s)

Time Elapsed 00:00:00.57

========== Build: 2 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========

Build: 0 errors, 6 warnings
```

### Example Project (If Possible)

https://github.com/SkyLabs-Innovation-Group/SkyDrop

--- a/Make.versions
+++ b/Make.versions
@@ -66,11 +66,11 @@ MAC_PACKAGE_VERSION=8.11.0.$(MAC_COMMIT_DISTANCE)
 # WARNING: Do **not** use versions higher than the available Xcode SDK or else we will have issues with mtouch (See https://github.com/xamarin/xamarin-macios/issues/7705)
 # When bumping the major macOS version in MACOS_NUGET_VERSION also update the macOS version where we execute on bots in jenkins/Jenkinsfile (in the 'node' element)
 
-IOS_NUGET_VERSION=15.4.302
-TVOS_NUGET_VERSION=15.4.302
-WATCHOS_NUGET_VERSION=8.5.302
-MACOS_NUGET_VERSION=12.3.302
-MACCATALYST_NUGET_VERSION=15.4.302
+IOS_NUGET_VERSION=15.4.303
+TVOS_NUGET_VERSION=15.4.303
+WATCHOS_NUGET_VERSION=8.5.303
+MACOS_NUGET_VERSION=12.3.303
+MACCATALYST_NUGET_VERSION=15.4.303
 
 
 # Defines the default platform version if it's not specified in the TFM. The default should not change for a given .NET version:

--- a/mk/xamarin.mk
+++ b/mk/xamarin.mk
@@ -7,7 +7,7 @@ MONO_BRANCH    := $(shell cd $(MONO_PATH) 2> /dev/null && git symbolic-ref --sho
 endif
 
 ifdef ENABLE_XAMARIN
-NEEDED_MACCORE_VERSION := cf9f7409e9a79d0d0835863cc9f5afe77ba5e74a
+NEEDED_MACCORE_VERSION := 29a1c1382e005e568b57d1b45e3877c8081dfc8b
 NEEDED_MACCORE_BRANCH := main
 
 MACCORE_DIRECTORY := maccore

